auction-server/src/auction/service/auction_manager.rs

```diff
@@ -180,6 +188,8 @@ impl AuctionManager<Evm> for Service<Evm> {
             )
             .await?;
 
+        tracing::Span::current().record("simulation_result", format!("{:?}", simulation_result));
```

Comment:
it seems that the default span attribute value length limit is around 2048 bytes. Maybe we should use a more compact representation here?

Reply:
I tested it with a string with 1million chars. It worked :-?

Reply:
this is specific for tempo which we use in staging and prod.
https://grafana.com/docs/tempo/latest/configuration/#ingestion-limits
`max_label_value_length`

I don't think it's good practice to store big values here in general.

Reply:
I think 2048 bytes is more than enough for these logs :-?